src/deploy-cromwell-on-azure/Deployer.cs

```diff
@@ -139,46 +151,74 @@ public async Task<int> DeployAsync()
                     if (configuration.Update)
                     {
                         resourceGroup = await azureSubscriptionClient.ResourceGroups.GetByNameAsync(configuration.ResourceGroupName);
+                        configuration.RegionName = resourceGroup.RegionName;
 
                         var targetVersion = Utility.DelimitedTextToDictionary(Utility.GetFileContent("scripts", "env-00-coa-version.txt")).GetValueOrDefault("CromwellOnAzureVersion");
 
                         ConsoleEx.WriteLine($"Upgrading Cromwell on Azure instance in resource group '{resourceGroup.Name}' to version {targetVersion}...");
 
                         var existingVms = await azureSubscriptionClient.VirtualMachines.ListByResourceGroupAsync(configuration.ResourceGroupName);
+                        var existingAksCluster = await ValidateAndGetExistingAKSClusterAsync();
+                        configuration.UseAks = existingAksCluster != null;
```

Comment:
Can we avoid  calling this task Validate and Get if the aks cluster name is nor present instead of calling and looking for its value. 
configuration.UseAks =  !string.IsNullOrEmpty(configuration.AksClusterName)

if (configuration.UseAks)
{
    await ValidateAndGetExistingAKSClusterAsync();
}